scripts/zkai_matched_d64_d128_evidence_table_gate.py

```
@@ -0,0 +1,1081 @@
+    return descriptor
+
+
+def _parse_reported_size_bytes(value: str) -> int:
+    parts = value.strip().split()
+    if len(parts) != 2 or parts[1] != "KB":
+        raise MatchedD64D128EvidenceTableError(f"unsupported reported proof size: {value}")
+    try:
+        kilobytes = float(parts[0])
+    except ValueError as err:
+        raise MatchedD64D128EvidenceTableError(f"unsupported reported proof size: {value}") from err
+    return int(round(kilobytes * 1000))
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)